package.json

```diff
@@ -86,7 +86,7 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "npm-run-all --parallel test:development:web-test-runner test:development:web-test-runner:serve-coverage",
-    "test:development:playwright": "chokidar './src/*test.visuals*' --initial --silent --command 'playwright test --ignore-snapshots --quiet --only-changed'",
+    "test:development:playwright": "chokidar './src/*test.visuals*' --initial --silent --command 'playwright test --quiet --only-changed --update-snapshots'",
```

Comment:
Turns out it's also useful to be able to inspect snapshots locally as you're writing test. Who would have known! 😆